Library/Formula/remarshal.rb

```diff
@@ -0,0 +1,62 @@
+require "language/go"
+
+class Remarshal < Formula
+  homepage "https://gowalker.org/github.com/dbohdan/remarshal"
```

Comment:
The homepage need to be changed to https://github.com/dbohdan/remarshal". Thanks.